roles/preflight/defaults/main.yml

```diff
@@ -8,4 +8,6 @@ preflight_podman_ca: ""
 catalog_url: "https://catalog.redhat.com/api/containers/v1"
 preflight_test_certified_image: false
 preflight_run_health_check: true
+do_container_parallel_test: false
```

Comment:
I don't think this is the best place to have this variable defined. This should be on `dci-openshift-app-agent`, as commented in the change you have opened on Gerrit.